classes/ActionScheduler_Versions.php

```diff
@@ -115,6 +115,58 @@ public static function initialize_latest_version() {
 	 * @return string
 	 */
 	public function active_source() {
+		$file   = __FILE__;
+		$dir    = __DIR__;
+		$path   = $this->active_source_path();
+		$themes = (array) search_theme_directories();
+
+		foreach ( $themes as $slug => $data ) {
+			$needle = trailingslashit( $data['theme_root'] ) . $slug . '/';
+
+			if ( 0 !== strpos( $file, $needle ) ) {
+				continue;
+			}
+
+			$theme = wp_get_theme( $slug );
+
+			if ( ! is_object( $theme ) || ! is_a( $theme, \WP_Theme::class ) ) {
+				continue;
+			}
+
+			return array(
+				'type' => 'theme',
+				'name' => $theme->Name,
```

Comment:
```suggestion
				'name' => $theme->name,
```

Seems like this should be lowercase.

Reply:
You'd think so, but no:
https://developer.wordpress.org/reference/classes/wp_theme/#properties.

Reply:
Both work, in fact, but yep that seems like the correct one to use (initially it wasn't working for me, which is why I flagged it ... must have been a temporary blip).